spec/features/charity_admin_can_manage_loan_spec.rb

```diff
@@ -7,21 +7,30 @@
       charity = create(:charity)
       role = Role.create(name: "charity_admin")
       UserRole.create(user: charity_admin, charity: charity, role: role)
-      family_1, family_2, family_3 = create_list(:family, 3, charity: charity)
+      family_1, family_2 = create_list(:family, 2, charity: charity)
       loan = create(:loan, family_id: family_1.id)
 
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return( charity_admin )
 
       visit admin_dashboard_path
+
       expect(page).to have_link("All Families")
 
       click_link "All Families"
 
-      expect(page).to have_content("#{family_1.last_name}, #{family_1.first_name}")
-      expect(page).to have_content("#{family_2.last_name}, #{family_2.first_name}")
-      expect(page).to have_content("#{family_3.last_name}, #{family_3.first_name}")
-      expect(page).to have_button("Update Loan")
-      expect(page).to have_button("Add Loan")
+      within(".row-nationality-active") do
+        expect(page).to have_content("#{family_1.last_name}, #{family_1.first_name}")
+        expect(page).to have_content("#{family_2.last_name}, #{family_2.first_name}")
+      end
+
```

Comment:
Good. Definitely like seeing this in the tests. We should make sure we add this wherever it's needed. Thanks for taking the time to do it.